packages/recommend-core/src/utils/mapToRecommendations.ts

```diff
@@ -1,8 +1,16 @@
 import { ProductRecord } from '../types';
-import { AverageIndexes } from '../types/AverageIndexes';
+import { AverageIndices } from '../types/AverageIndices';
 import { IndexTracker } from '../types/IndexTracker';
 
-import { getAverageIndexes } from './computeAverageIndexes';
+import { getAverageIndices } from './computeAverageIndices';
+
+/**
+ * Returns the reordered list of recommendations based on average indices.
+ *
+ * @param hits - recommendations
+ * @param maxRecommendations - max number of recommendations
+ * @param nrOfObjs - total number of products for which we are retrieving recommendations (objectsIDs.length)
+ */
```

Comment:
I meant like this: https://github.com/algolia/recommend/blob/7d9bb18046f715e8bcf61bf84ae0dbf298ba095a/packages/recommend-core/src/getTrendingItems.ts#L7-L20